tests/core_tests.py

```diff
@@ -156,6 +156,24 @@ def test_slices(self):
             print("[{name}]/[{method}]: {url}".format(**locals()))
             self.client.get(url)
 
+    def test_slices_V2(self):
```

Comment:
this test is passing, but for some reason `test_client (tests.druid_tests.DruidTests)` is now failing. need to investigate.